### ISSUE TITLE
CPU at 100% when network or source is down

### ISSUE THREAD

**Issue by thibaultandrieux** [Simonmicro/better-backgrounds#13]
I get 100% CPU utilization in htop whenever there is a network problem while better-backgrounds tries to fetch a new background from a source, even after a notification shows error message. I then remove better-backgrounds from taskbar to stop CPU utilization.

Could better-backgrounds just stop trying after the first fail ? Or wait some time before retrying.

**Comment by simonmicro** [comment]
Oh, I have not noticed that. I guess the icon animation is the problem... Maybe you could disable it until I fixed this and report back. I'll take a look into this later!

**Comment by simonmicro** [comment]
I guess you noticed it, because Unsplash had problems today? :smiling_imp: :laughing:

**Comment by thibaultandrieux** [comment]
Yes the problem arises with Unsplash and/or when I activate a VPN that does
not allow access to Unsplash.
Your suggested workaround (disabling the icon animation) is working nicely,
so yes I suppose the problem resides with the icon animation in the taskbar.

**Comment by simonmicro** [comment]
Yes, I can also confirm the problem is definitively the Tweener animation of the icon. The problem seems to lay way deeper than I can change. I'll look into other, may not such "demanding" animations...

**Comment by simonmicro** [comment]
Okay. Done. I was able to reduce the CPU load by up to - hold your pants - 8%. The main problem is that something deep inside Cinnamon thinks that rendering the panels on a software level is a good idea and I can't convince it to use the GPU when available. Even when I attempt to skip every animation frame the CPU is still working (idk why). At this point I just changed the animation to something softer and will roll out the update in the next days. The final solution: Disable the animations when they disturb you. I am sorry I can't "fix" it better for now.
